linode_api4/objects/object_storage.py

```diff
@@ -63,13 +87,10 @@ def make_instance(cls, id, client, parent_id=None, json=None):
         Override this method to pass in the parent_id from the _raw_json object
         when it's available.
         """
-        if json is None:
-            return None
-
-        cluster_or_region = json.get("region") or json.get("cluster")
-
-        if parent_id is None and cluster_or_region:
-            parent_id = cluster_or_region
+        if json is not None:
+            cluster_or_region = json.get("region") or json.get("cluster")
+            if parent_id is None and cluster_or_region:
+                parent_id = cluster_or_region
```

Comment:
optional: This snippet can be simplified a bit 🙂 

```suggestion
        if json is not None:
            parent_id = parent_id or json.get("region") or json.get("cluster")
```

Reply:
Nice simplification!